indico/modules/events/contributions/models/contributions.py

```diff
@@ -331,6 +355,9 @@ def get_non_inheriting_objects(self):
     def get_field_value(self, field_id):
         return next((v.friendly_data for v in self.field_values if v.contribution_field_id == field_id), '')
 
+    def get_field(self, field_id):
```

Comment:
Would you agree in renaming the `get_field_value` to `get_field_friendly_value`, so that I can rename the `get_field` to `get_field_value`?